src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiry-notification/pwd-expiry-notification.component.html

```diff
@@ -0,0 +1,10 @@
+<alert class="no-margin-bottom" type="{{ alertType }}" *ngIf="expiryDays <= levels[warningLevel]" [dismissible]="true" i18n>
```

Comment:
Please format the html to have only 1 attr per line.

This is supposed to be catch by the html linter, but that is not working properly.
I have submitted a PR to fix this: #30998